app/models/ability.rb

```diff
@@ -0,0 +1,71 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+
+    can :manage, :all if user.is_admin?
+    can [:manage_item, :manage_content, :manage_desc_metadata, :register_apo], :all if user.is_manager?
+    can [:view_item, :view_metadata, :view_content], :all if user.is_viewer?
```

Comment:
I need to understand how this provides the precedence for Argo `is_admin`, `is_manager` and `is_viewer` over the object permissions.  Does the `can?` logic apply `a || b` or is it `a && b`?  We mostly want `a || b` and when it's something like `is_admin || object.can_manage_item(user.roles(apo.pid))` we don't want it to evaluate the object permission.

We should discuss to clarify the differentiation of `is_admin` from `is_manager`.